core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sklearn/SklearnPredictionOpDesc.scala

```diff
@@ -38,7 +38,9 @@ class SklearnPredictionOpDesc extends PythonOperatorDescriptor {
        |            input_features = tuple_
        |            if "$groundTruthAttribute" != "":
        |                input_features = input_features.get_partial_tuple([col for col in tuple_.get_field_names() if col != "$groundTruthAttribute"])
-       |            tuple_["$resultAttribute"] = str(self.model.predict(Table.from_tuple_likes([input_features]))[0])
+       |                tuple_["$resultAttribute"] = type(tuple_["$groundTruthAttribute"])(self.model.predict(Table.from_tuple_likes([input_features]))[0])
```

Comment:
Do we need such a cast? Can we just use the prediction output?

Reply:
It is a 'must' because Python's prediction can return a type that cannot be handled by Scala, such as numpy.int64.

Reply:
Ok